src/runtime/runtime-debug.cc

```diff
@@ -1014,8 +1014,33 @@ RUNTIME_FUNCTION(Runtime_RecordReplayAssertExecutionProgress) {
     CHECK(gRecordReplayHasCheckpoint);
   }
 
-  recordreplay::Assert("ExecutionProgress %zu %s:%d:%d",
-                       *gProgressCounter, name.c_str(), info.line + 1, info.column);
+  recordreplay::Assert(
+      "JS ExecutionProgress PC=%zu scriptId=%d @%s:%d:%d (%d %d %d %d)",
+      *gProgressCounter, script->id(), name.c_str(), info.line + 1, info.column,
+      recordreplay::AreEventsDisallowed(),
+      !recordreplay::HasDivergedFromRecording(),
+      function->shared().IsUserJavaScript(),
+      function->shared().HasSourceCode());
```

Comment:
What is the point of including this in all these assertions?  These seem to be random bits of V8 state and not relevant to the purpose of the assertion.